lib/systems/default.nix

```diff
@@ -370,6 +370,22 @@ let
         # Handle Android SDK and NDK versions.
         androidSdkVersion = args.androidSdkVersion or null;
         androidNdkVersion = args.androidNdkVersion or null;
+
+        zigTarget =
+          let
+            cpu = final.parsed.cpu.name;
+            os =
+              if final.isMacOS then
+                "macos" # Instead of "darwin"
+              else
+                final.parsed.kernel.name;
+            abi =
+              {
+                "unknown" = "none"; # e.g. x86_64-macos-none
+              }
+              .${final.parsed.abi.name} or final.parsed.abi.name;
+          in
+          args.zigTarget or "${cpu}-${os}-${abi}";
```

Comment:
for macOS we should put here: `.${stdenv.targetPlatform.darwinSdkVersion}`?

Reply:
I don't understand this question. The MacOS targets for Zig are `{aarch64,x86_64}-macos-none`

Reply:
I mean, other nixpkgs libraries are compiled with darwinSdkVersion=11.3, but zig will compile with 26.0 (getting this from host probably)

Reply:
Honestly I have no clue